src/wwwroot/css/themes/Beweish.css

```diff
@@ -0,0 +1,391 @@
+    --shadow-md: 0 4px 6px rgba(0, 0, 0, 0.4);
+    --shadow-lg: 0 10px 15px rgba(0, 0, 0, 0.5);
+}
+
+/* Button styling */
+.basic-button {
+    background-color: var(--button-background);
+    border: 1px solid var(--border-color);
+    color: var(--button-text);
+    transition: all 0.3s ease;
+    box-shadow: var(--shadow-sm);
+}
+
+.basic-button:hover {
+    background-color: var(--button-background-hover);
+    border-color: var(--emphasis);
+    box-shadow: var(--shadow-md);
+}
+
+.basic-button:active {
+    background-color: var(--emphasis);
+}
+
+/* Input groups and dividers */
+.input-group {
+    border: 1px solid var(--border-color);
+    background-color: var(--background-panel);
+    margin-bottom: 1rem;
+    border-radius: 4px;
+    overflow: hidden;
+    box-shadow: var(--shadow-sm);
+}
+
+.group-label {
+    background-color: var(--background-gray);
+    border-bottom: 1px solid var(--border-color);
+    padding: 0.5rem;
+    color: var(--text);
+}
+
+/* Custom scrollbar */
+::-webkit-scrollbar {
+    width: 8px;
+    height: 8px;
+}
+
+::-webkit-scrollbar-track {
+    background: var(--background);
+}
+
+::-webkit-scrollbar-thumb {
+    background: var(--emphasis);
+    border-radius: 4px;
+}
+
+::-webkit-scrollbar-thumb:hover {
+    background: var(--emphasis-soft);
+}
+
+/* Card styling */
+.card {
+    background: var(--background-panel);
+    border: 1px solid var(--border-color);
+    box-shadow: var(--shadow-sm);
+}
+
+.card:hover {
+    box-shadow: var(--shadow-md);
+}
+
+/* Input styling */
+input, select, textarea {
+    background-color: var(--background);
+    border: 1px solid var(--border-color);
+    color: var(--text);
+    transition: all 0.3s ease;
+}
+
+input:focus, select:focus, textarea:focus {
+    border-color: var(--emphasis);
+    outline: none;
+    box-shadow: var(--shadow-sm);
+}
+
+/* Link styling */
+a {
+    color: var(--emphasis);
+    transition: color 0.3s ease;
+}
+
+a:hover {
+    color: var(--emphasis-soft);
+}
+
+/* Modal styling */
+.modal-content {
+    background: var(--background-panel);
+    border: 1px solid var(--border-color);
+    box-shadow: var(--shadow-lg);
+}
+
+.modal-header {
+    border-bottom: 1px solid var(--border-color);
+}
+
+.modal-footer {
+    border-top: 1px solid var(--border-color);
+}
+
+/* Progress bar */
+.progress {
+    background: var(--background-gray);
+}
+
+.progress-bar {
+    background: var(--emphasis);
+}
+
+/* Alert styling */
+.alert-success {
+    background: var(--green);
+    color: var(--text);
+}
+
+.alert-warning {
+    background: #ffbb00;
+    color: var(--text);
+}
+
+.alert-danger {
+    background: var(--red);
+    color: var(--text);
+}
+
+.alert-info {
+    background: var(--text-soft);
+    color: var(--background);
+}
+
+/* Splitter styling */
+.splitter-bar {
+    background-color: var(--border-color) !important;
+}
+
+.splitter-bar:hover {
+    background-color: var(--emphasis) !important;

```

Comment:
why are there *so many things* in this css?

Like this line for example, is setting the color of splitter-bar to what it already is but with `!important` tacked on? wtf?